server/src/test/java/org/elasticsearch/common/time/DateUtilsRoundingTests.java

```diff
@@ -46,4 +46,14 @@ public void testDateUtilsRounding() {
             }
         }
     }
+
+    public void testIsLeapYear() {
```

Comment:
This seems like a good candidate for randomized testing, since we have an alternative (slower) implementation that the new implementation should match exactly.  Then again, that might be overkill.

Reply:
We sort of have that above. I wrote this to have something simple to double check my bit twiddling.

Reply:
ah, so we do.  Should have looked deeper before I commented.